brave/brave-ios
# Fix #6813: Remove usage of LocalRequestHandler in internal scripts

## Security Review
- https://github.com/brave/security/issues/1169

## Summary of Changes
- Remove usage of LocalRequestHandler in scripts, in favour of having the page reload itself via `location.replace`


This pull request fixes #6813

## Submitter Checklist:


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`

## Test Cases:
1. Enter Reader-Mode and make sure it works.
2. Exit Reader-Mode and make sure it works.
3. Enter and Exit Reader-Mode on a very bad network, if it shows `loading screen` it is working.
